src/device-registry/.env

```diff
@@ -1,3 +1,5 @@
 PROJECT_ID=airqo-250220
```

Comment:
This file should not be part of the commits made. Utilize .gitignore file for this or investigate in case it is already there.